server/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+const app = require('../app');
+const debug = require('debug')('server:server');
+const http = require('http');
```

Comment:
Why `http` server when you have already included the `express`?

Reply:
Initially, I was using the express generator for the server-side repository, so it was generated code. Now I have made the server-side repository structure according to our need.